resqpy/time_series/time_series.py

```diff
@@ -0,0 +1,158 @@
+"""time_series.py: RESQML time series class."""
```

Comment:
I would rather this module was called _time_series_functions.py or _time_series_fns.py or just _functions.py if you prefer.